src/Security/CookiePolicy/src/HttpOnlyPolicy.cs

```diff
@@ -3,9 +3,21 @@
 
 namespace Microsoft.AspNetCore.CookiePolicy
 {
+    /// <summary>
+    /// Describes the HttpOnly behavior for cookies.
+    /// </summary>
     public enum HttpOnlyPolicy
     {
+        /// <summary>
+        /// The cookie does not have a configured HttpOnly behavior. This cookie can be accessed by
+        /// JavaScript <c>document.cookie</c> API.
```

Comment:
`No policy is applied, response cookies are allowed to be sent with or without the HttpOnly attribute.`